# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/coreutils-finitel.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- coreutils-6.12/src/seq.c.orig	2008-05-26 08:40:32.000000000 +0200
+++ coreutils-6.12/src/seq.c	2008-11-01 12:23:03.000000000 +0100
@@ -29,6 +29,11 @@
 #include "quote.h"
 #include "xstrtod.h"
 
+/* uClibc <math.h> uses __finite(l) but does not provide it in -lm */
+#ifdef __UCLIBC__
+#undef isfinite
+#endif
+
 /* Roll our own isfinite rather than using <math.h>, so that we don't
    have to worry about linking -lm just for isfinite.  */
 #ifndef isfinite
